packages/swing-store/src/index.js

```diff
@@ -5,7 +5,7 @@ export { importSwingStore } from './importer.js';
 export { makeArchiveSnapshot, makeArchiveTranscript } from './archiver.js';
 
 // for the benefit of tools like SwingSet/misc-tools/replay-transcript.js
-export { makeKVStore, getKeyType } from './kvStore.js';
+export { getKVStoreKeyType } from './util.js';
```

Comment:
I didn't realize we exported this from the package, but similarly, we may want to re-export a second time as the old name to avoid unnecessary export changes.

```suggestion
export { getKVStoreKeyType, getKVStoreKeyType as getKeyType } from './util.js';
```